_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Ship Prometheus metrics from AWS CloudWatch with the OpenTelemetry Collector
+logo:
+  logofile: prometheusio-icon.svg
```

Comment:
We need to use cloudwatch instead of Prometheus.

![download](https://user-images.githubusercontent.com/48534558/104850586-40f31580-58f0-11eb-8894-46de35156113.png)

Reply:
Done in next commit